golang/content.md

```diff
@@ -59,7 +59,7 @@ Alternatively, you can build for multiple platforms at once:
 $ docker run --rm -it -v "$PWD":/usr/src/myapp -w /usr/src/myapp %%IMAGE%%:1.8 bash
 $ for GOOS in darwin linux; do
 >   for GOARCH in 386 amd64; do
->     go build -v -o myapp-$GOOS-$GOARCH
+>     GOOS=$GOOS GOARCH=$GOARCH go build -v -o myapp-$GOOS-$GOARCH
```

Comment:
Oh good catch, these need to be exported.  In the interest of shorter line length and making it more obvious what's happening here, I think I'd prefer to go for an explicit `export`, ala:

```suggestion
>     export GOOS GOARCH
>     go build -v -o myapp-$GOOS-$GOARCH
```

Reply:
Oh, what you said is right! But i think  this change the global env GOOS and GOARCH, maybe  we should unexport then in the end.

Reply:
This is just an example, so I think that's best left as an exercise for the reader.

Reply:
Okey,  i write as u recommened, maybe i think too more, ^__^